front/src/Phaser/Login/SelectCharacterScene.ts

```diff
@@ -26,6 +26,7 @@ export const SelectCharacterSceneName = "SelectCharacterScene";
 export class SelectCharacterScene extends AbstractCharacterScene {
     protected selectedWoka!: Phaser.GameObjects.Sprite | null; // null if we are selecting the "customize" option
     protected playerModels!: BodyResourceDescriptionInterface[];
+    protected currentSelectedItem!: WokaSlot;
```

Comment:
We probably don't need a member variable for that. Your variable is only used locally it seems.